main.js

```diff
@@ -150,7 +153,7 @@
         .then((data) => {
             const records = data.result.records;
             records.forEach((record, i) => {
-
+                var popup = JSONtoTable(record);
```

Comment:
For clarity, maybe call this `popupHtml`? Alternatively, I think it would be clear enough if `JSONtoTable(record)` was just nested in the `bindPopup()` call.